JAVALeetCode/Easy/CanPlaceFlowers605/CanPlaceFlowers605.java

```diff
@@ -8,19 +8,19 @@ public boolean canPlaceFlowers(int[] flowerbed, int n) {
             n_temp++;
             flowerbed[0]=1;
         }
-        for(int i = 0; i < flowerbed.length - 2 ; i ++) {
```

Comment:
Plz do not change iterator variable. Instead change flowerbed to flowerBed.

Reply:
Ok, I will do it.